crates/matrix-sdk-base/src/event_cache/store/media/media_retention_policy.rs

```diff
@@ -12,10 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Configuration to decide whether or not to keep media in the cache, allowing
+//! to do periodic cleanups to avoid to have the size of the media cache grow
+//! indefinitely.
```

Comment:
Explanation how to run a clean up, or when it is done if done automatically, would be particularly helpful!

Reply:
I added instructions for this and a sentence about automatic periodic cleanups.